include/dos_system.h

```diff
@@ -93,6 +93,11 @@ struct FileStat_Block {
 
 class DOS_DTA;
 
+struct DOS_Filename {
```

Comment:
The MODULE_Function naming scheme is really only for API calls. For structs, we drop the module prefix and go with a sensible name.

In this case, the sensible name would still be `DosFilename`, just it would be all camel case like the rest of the structs.

Reply:
Renamed